tests/fixtures/simple_app/node_modules/subfolder/file.rb

```diff
@@ -0,0 +1 @@
+# some ignored file
```

Comment:
This file was accidentally removed at some point, so it's added back to make the test above pass correctly.